src/emitter.rs

```diff
@@ -260,6 +252,26 @@ impl<'a> YamlEmitter<'a> {
         }
     }
 
+    fn emit_literal_block(&mut self, v: &String) -> EmitResult {
+        let ends_with_newline = v.chars().last() == Some('\n');
```

Comment:
Over in https://github.com/Ethiraric/yaml-rust2/pull/32/files#diff-687937e1c52b6e0080c70ccc9bdde14a35f5dfdfa6d0105c508b19181cafde6fR254 it's using `v.ends_with('\n')` instead, which is better.

Reply:
We're using a mutable `&String` parameter here. Use a `v: &str` borrow instead.

Reply:
Fixed.